wkt.build.js

```diff
@@ -297,6 +297,10 @@ function d2r(input) {
 }
 
 function cleanWKT(wkt) {
+  if (wkt.AUTHORITY) {
+    var authority = Object.keys(wkt.AUTHORITY)[0];
+    wkt.title = `${authority}:${wkt.AUTHORITY[authority]}`;
+  }
```

Comment:
This file is auto-built, you should not be making changes to it. We'll need to make this clear in the README and/or with a proper .gitignore.

Reply:
Also, I don't think it makes sense to change the library to match its 3rd party type definition. It should be the other way around. Which means @types/proj4 should be changed to match what we have here.

I could be convinced otherwise if there is a strong argument for having a title that shows the authority. I can't find one myself.

Reply:
Based on the 3rd party type definition title was the only obligatory property, thus It was the only one on wich I could relied as an existing id, And follwing EPSG.io, I fell like title in format 'EPSG:XXXX' seemed like a good identifier.

What property should I use to id a specifique parsed wkt ?

Reply:
The @types/proj4 types tell fair tales. `title` is neither used for `wkt` nor for `proj` definitions. If you're using wkt-parser standalone, `name` at the root level is what you should be looking at. As far as I know, the AUTHORITY entry is not mandatory in the WKT v1 spec, so your pull request would cause errors for `wkt` defs that don't contain one.

Reply:
Thanks for the clarification about the @types/proj4.
I was using the name field but to avoid doublon between a proj.def defined through it's name and the same one defined using EPSG:XXXX, i thought about trying to get a more unique identifier.

Just to clarify I don't understand how my PR would cause errors as it's only if there is an AUTHORITY field that i set the property title from it.

Reply:
It would cause errors or at least unexpected output if what you're putting in the `title` string is not available. So I'd prefer something like
```js
if (wkt.AUTHORITY) {
    var authority = Object.keys(wkt.AUTHORITY)[0];
    if (authority && authority in wkt.AUTHORITY) {
      wkt.title = `${authority}:${wkt.AUTHORITY[authority]}`;
    }
  }

Reply:
I added what you sugest to avoid the case with an empty wkt.AUTHORITY.